custom_components/tahoma/const.py

```diff
@@ -76,7 +76,6 @@
     "HitachiAirToAirHeatPump": CLIMATE,  # widgetName, uiClass is HitachiHeatingSystem (not supported)
     "HumiditySensor": SENSOR,
     "Light": LIGHT,
-    "LightSensor": SENSOR,
```

Comment:
TODO: don’t warn when LightSensor detected.

Reply:
At the end, we should ignore all the SENSOR devices. But yes, for the moment we can add it the ignore list.